tests/framework/utils/snapshot_helper.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
nit: let's name the file `snapshot.go`. The other files in this package don't need the `_helper` suffix.